.github/workflows/verify.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: [push]
+
+jobs:
+  verify:
```

Comment:
Any good idea for a name?

<img width="400" alt="Bildschirmfoto 2020-01-29 um 15 31 49" src="https://user-images.githubusercontent.com/26137398/73365415-84520f80-42ac-11ea-836c-ead97d399464.png">

Reply:
/shrug I think verify is good enough, isn't it?

Reply:
agree ...

Reply:
We could think of defining dedicated job names (`step generation`, `format`, `dependencies`), that way one could directly see here which check failed.